compiler/rustc_feature/src/accepted.rs

```diff
@@ -142,6 +142,8 @@ declare_features! (
     (accepted, dyn_trait, "1.27.0", Some(44662), None),
     /// Allows integer match exhaustiveness checking (RFC 2591).
     (accepted, exhaustive_integer_patterns, "1.33.0", Some(50907), None),
+    /// Allows explicit generic arguments specification with `impl Trait` present.
+    (accepted, explicit_generic_args_with_impl_trait, "1.62.0", Some(83701), None),
```

Comment:
Bootstrap occurs the Friday before a release, I think. If this is the case, it won't land until 1.63. No problem if you'd prefer to wait to be certain, though.

Reply:
Heh, yeah, I thought this was optimistic, but I didn't want to screw myself in case it did get reviewed in time :-)

Reply:
The only reason I brought it up is because FCP won't even complete until after then. But I presume you already knew that; it's not like you're new.